extension/page-detect.js

```diff
@@ -33,6 +33,17 @@ window.pageDetect = (() => {
 
 	const isBlame = () => isRepo() && /^\/blame\//.test(getRepoPath());
 
+	const getOwnerAndRepo = () => {
```

Comment:
Not caching these values in the scope of the module, because it'll end up making testing more difficult (#148).